website/www/site/content/en/contribute/_index.md

```diff
@@ -24,366 +24,177 @@ See the License for the specific language governing permissions and
+    <td class="list-text-size-md">
+      <ul>
+        <li>Create and contribute a custom <a href="https://beam.apache.org/documentation/programming-guide/#pipeline-io">I/O connector</a> for a specific data storage system you want to connect to, if it is not on <a href="https://beam.apache.org/documentation/io/built-in/">the I/O list yet</a>.</li>
+        <li>Look at our project backlog to get some inspiration on what to build.</li>
+        <li>Follow our <a href="https://beam.apache.org/contribute/">Code Contribution Guide</a>.</li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>Share a code sample or template</td>
+    <td class="list-text-size-md">
+      <ol>
+        <li>Read our <a href="https://beam.apache.org/contribute/">Code Contribution Guide</a>.</li>
+        <li>Create your code sample or template. </li>
+        <li><a href="https://beam.apache.org/contribute/#make-your-change">Make a Pull Request</a> to add it to the Beam example folder.</li>
+      </ol>
+    </td>
+  </tr>
+  <tr>
+    <td>Test a release candidate</td>
+    <td>
+      Anybody can <a href="https://beam.apache.org/contribute/release-guide/">propose a release</a> via the dev@beam.apache.org mailing list. Try Apache Beam releases in your projects, vote for release candidates, inform the community about the results and any issues found via <a href="mailto:dev@beam.apache.org">dev@beam.apache.org</a>. Learn more about how to validate a Beam release <a href="https://beam.apache.org/blog/validate-beam-release/">here</a>.
+    </td>
+  </tr>
+  <tr>
+    <td>Other ideas?</td>
+    <td>
+      If you want to contribute in another way and you’re not sure how to get started, send a proposal to <a href="mailto:dev@beam.apache.org">dev@beam.apache.org</a>.
+    </td>
+  </tr>
+</table>
+
+
+## Become a Committer
+
+If you like contributing to Apache Beam, you are active in the Apache Beam mailing lists, Slack channel, or you attend/organize events, then you are a candidate to become a project committer.
+
+Committers are community members that have write access to the project’s repositories  (i.e., they can modify the code, documentation, and website by themselves and also assist with code reviews and accept other contributions).
+
+Even though there is no strict protocol to become a committer, the most important thing we look for is that you display the traits of an Apache Beam committer and that you have been consistently contributing to the project and community.
+
+Candidates for new committers are suggested by current committers or PMC members, and voted upon by the PMC team. So if you would like to become a committer, engage with the community and keep contributing to Apache Beam, you might also want to talk to other committers and ask for their advice and guidance.  Feel free to email any committer directly and ask for their advice!
+
+## The Traits of an Apache Beam Committer
+
+{{< contributor/row_of_traits committer_traits >}}
```

Comment:
This section looks like:

<img width="964" alt="image" src="https://user-images.githubusercontent.com/26118465/159727714-037e5866-d89b-45c8-ba7f-f1e9c729ef0f.png">

Could we change style of the `Learn more.`, so it looks like in [Case Studies Page](https://beam.apache.org/case-studies/hop/)

<img width="125" alt="image" src="https://user-images.githubusercontent.com/26118465/159728098-d98107ac-f494-4a17-834e-5e37f5e38c2f.png">

Reply:
done